### PR TITLE
refactor: minimize cookie banner

### PR DESCRIPTION
**Pull request by SGrueber**
## PR Type

[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no API changes)
[ ] Build-related changes
[ ] CI-related changes
[ ] Documentation content changes
[ ] Application / infrastructure changes
[x] Other:  Performance

## Introduction

The cookie banner can have a significant impact on your CVW metrics, especially if it contains the largest block of content in the viewport.  If Google’s bots consider your banner’s element the largest, your Largest Contentful Paint (LCP)  score will depend on how fast your banner loads. However, because the cookie banner is not rendered on the server-side, speeding up the cookie banner would be very tough. That's why it is recommended to make the banner as small as possible.

## What Is the Current Behavior?

The cookie banner is the largest element on many pages. This has a bad impact of the LCP metrics of the CWV.

Issue Number: Closes #

## What Is the New Behavior?
The cookie banner has been minimized: 
- The title has been removed
- The text has been shortened
- The 'Accept only necessary' button has been removed
- Tool tips have been added to the buttons to explain a bit more their functionality
- The styling has been adapted, spaces have been reduced

On desktop sites and the most mobile sites the cookie banner is not the largest element any more.

## Does this PR Introduce a Breaking Change?

[ ] Yes
[x] No

## Other Information

[AB#94565](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/94565)

### CODE REVIEW

**Comment by M-Behr** [comment]
IAD: 
image

**Review by M-Behr — APPROVED** [review]
**Good work!** :)

**Review by iwiederhold — APPROVED** [review]
YES! THX

File: src/assets/i18n/de_DE.json [hunk 1]
```diff
@@ -836,9 +836,10 @@
   "cookie.consent.option.tracking.name": "Tracking und Performance",
   "cookie.preferences.heading": "Cookie-Einstellungen",
   "cookies.banner.accept_all": "Alle akzeptieren",
-  "cookies.banner.accept_only_required": "Nur notwendige akzeptieren",
+  "cookies.banner.accept_all.tooltip": "Stimmen Sie der Verwendung aller Cookies zu",
```
Comment (mglatter) [inline src/assets/i18n/de_DE.json:839]: What about "Der Verwendung aller Cookies zustimmen"?

**Comment by mglatter** [comment]
French will follow

**Review by mglatter — APPROVED** [review]

**Review by andreassteinmann — APPROVED** [review]

**Review by M-Behr — APPROVED** [review]
Great!

**Review by iwiederhold — APPROVED** [review]
YES! THX!

**Comment by SGrueber** [comment]
image

final implementation

**Pull request merged**